pkgs/games/steam-rom-manager/default.nix

```diff
@@ -0,0 +1,30 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "steam-rom-manager";
+  version = "2.3.40";
+  name = "${pname}";
+
+  src = fetchurl {
```

Comment:
This doesn't have to be in the `let` block - why not include it in `appimageTools.wrapType2` instead and skip the `inherit src` part.